### DISCUSSION TITLE
Stop or rework the "Action: AUR Publish - Development version"?

### DISCUSSION THREAD

**Discussion by Narrat** [ulises-jeremias/dotfiles#22] [Q&A] [answered]
Hello.
I noticed your aur repo [dots-git](https://aur.archlinux.org/packages/dots-git). And I noticed some behaviour which isn't really according to the [guidelines](https://wiki.archlinux.org/title/AUR_submission_guidelines).
Main point being it is a VCS package. Those package should only receive a version bump if there are breaking changes that [warrant](https://wiki.archlinux.org/title/AUR_submission_guidelines#Publishing_new_package_content) a build.
In your case it is kinda worse as the action is set to run the action twice a day? At least the aur repo gets updated that often. And most of the time with empty commits which pollutes the history of the repo quite significantly. Not made easier by the fact that everything has the same commit message and such.
So please change the process. Or if you doubt me get in touch with the Trusted Users via the [aur-general mailing list](https://lists.archlinux.org/mailman3/lists/aur-general.lists.archlinux.org/) and ask for confirmation if the behaviour is okay.

**Comment by ulises-jeremias** [comment, marked as answer]
hi! thanks for opening this discussion! will fix this asap during this week and will reach out to you for review as soon as the fix is ready. Thank you! 😊

**Comment by ulises-jeremias** [comment]
Fixed!

**Reply by Narrat** [reply]
Great that you could take care of this.
Although there still seems to be something odd with the github action? When you pushed your changes they were still followed by an empty commit?
And FabioLolix added a [comment](https://aur.archlinux.org/packages/dots-git#comment-918738) with additional remarks :)

**Reply by ulises-jeremias** [reply]
@Narrat after some research I was able to fix the issue about the empty commit [here](https://github.com/ulises-jeremias/dotfiles/blob/99e37eb4436a250939b97d6d564a7a90ec92683d/.github/workflows/aur-dev-publish.yml#L43)!
